### PR TITLE
Add Authentication controller with providing User entity without JWT

### PR DESCRIPTION
**Pull request by SkillfulDev**

### CODE REVIEW

File: src/main/java/online/carsharing/entity/User.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Table(name = "users")
+@Getter
+@Setter
+@RequiredArgsConstructor
+public class User {
```
Comment (azzilian) [inline src/main/java/online/carsharing/entity/User.java]: Question - Dont we need @ToString and @EqualsAndHashCode?

**Review by azzilian — COMMENTED** [review]
Gj! Only thing is my question - dont we need @tostring and @EqualsAndHashCode?
Also maybe for every entity in our project?

File: src/main/java/online/carsharing/entity/User.java (hunk 1 above)
Comment (SkillfulDev) [reply src/main/java/online/carsharing/entity/User.java]: In this code there are no use cases when we need to use it. That’s why they are not needed

**Review by nklimovych — DISMISSED** [review]

**Review by azzilian — DISMISSED** [review]
Everything is fine!

**Review by azzilian — APPROVED** [review]
Changes approved!

**Review by nklimovych — APPROVED** [review]

**Review by HAIOVYI — APPROVED** [review]

**Pull request merged**